### ISSUE TITLE
Warning of missing .env file

### ISSUE THREAD

**Issue by brandonjmckay** [fivetanley/ember-cli-dotenv#8]
I specified a `.env.development` style `.env`, one for each environment.

But when I ran the server, I got these warnings:

```
{ [Error: ENOENT, no such file or directory '/Users/bmckay/Rails/agent5/.env']
  errno: -2,
  code: 'ENOENT',
  path: '/Users/bmckay/Rails/agent5/.env',
  syscall: 'open' }
```

Seems like you shouldn't get this error unless there is an environment without a `.env` file defined, and that it should say something more to that effect. 
Possibly _only_ when running that environment as well.

This message got printed out many times, so it added a lot of noise.
As a workaround, I just added an empty `.env` file for now.

**Comment by poteto** [comment]
I have the same issue.

**Comment by blimmer** [comment]
Ditto.

**Comment by Kulja** [comment]
Same problem here.

**Comment by fivetanley** [comment]
Fixed by https://github.com/fivetanley/ember-cli-dotenv/pull/10

**Comment by fivetanley** [comment]
In 1.0.2

**Comment by bashmach** [comment]
Hi Stanley!

Your fix is not working.

`fs.existsSync` should be checked first in [this condition](https://github.com/fivetanley/ember-cli-dotenv/blob/master/index.js#L31).

I would have sent a PR to fix it, but I think it can be fixed with merge of #11.
